src/Xamarin.Android.Build.Tasks/Tasks/Aapt.cs

```diff
@@ -151,7 +151,15 @@ bool RunAapt (string commandLine)
 
 		bool ExecuteForAbi (string cmd, string currentResourceOutputFile)
 		{
-			var ret = RunAapt (cmd);
+			var output = new List<Tuple<string, bool>> ();
+			var ret = RunAapt (cmd, output);
+			foreach (var line in output) {
+				if (line.Item2) {
```

Comment:
...and this right here is why I'm not fond of `Tuple<string, bool>`. `line.stderr` would be *so much more readable*.

Not a chance I'm going to remember that `Item2` is whether the message came from stdout vs. stderr.